src/main/resources/scripts/jenkins/feature/build-dialog.soy

```diff
@@ -84,8 +84,8 @@
     <div id="jenkins-form-{$count}" class="field-group jenkins-form">
         <div class="checkbox">
             <input class="checkbox" type="checkbox" id="build-param-value-{$count}" name="build-param-value-{$count}"
-                {$value == 'true' ? 'checked="checked"' : ''}>
-            <label for="build-param-value-2">{$key}</label>
+                {$value === true ? 'checked="checked"' : ''}>
```

Comment:
This line isn't actually javascript code but the [soy templating language](https://developer.atlassian.com/server/confluence/writing-soy-templates-in-your-plugin/) (also called google closure templates). As such, the triple equality operator doesn't exist and should be a double equals:
```suggestion
                {$value == true ? 'checked="checked"' : ''}>
```

Reply:
Done & rebased, thanks for the catch